docs/hands-on-lab/02-connecting-to-the-lightbulbs.md

```diff
@@ -0,0 +1,28 @@
+# Connecting to the Lightbulbs
+
+While there are no technical restrictions on the kinds of remote devices or APIs you can connect to with transports, for the purpose of this workshop we are connecting to a Philips HUE hub and make some colourful wireless lightbulbs light up. If you (understandably) do not have physical devices available, you can get going with the Hue Emulator.
+
+## Hue Emulator
+
+Download the [HueEmulator-v0.8.jar](https://github.com/SteveyO/Hue-Emulator/blob/master/HueEmulator-v0.8.jar) from [SteveyO/Hue-Emulator](https://github.com/SteveyO/Hue-Emulator).
```

Comment:
Won't they have already done this in the installing section?

Reply:
that's a duplicate, yeah.